client/src/pages/Resource/index.jsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import { ResourceService, useService } from "../../services";
+
+import "./Resource.scss";
+
+export default function Resource() {
+	const [resource, setResource] = useState(undefined);
+	const resourceService = useService(ResourceService);
+	const { id } = useParams();
+
+	useEffect(() => {
+		resourceService.getOne(id).then(setResource);
+	}, [id, resourceService]);
+
+	if (!resource) {
+		return null;
+	}
+
+	return (
+		<>
+			<h2>
+				Resource &gt; <span>{resource.title}</span>
+			</h2>
+			<section>
+				<table aria-label="resource details" className="resource-details">
```

Comment:
The table uses `aria-label` which is good, but how about adding more descriptive labels for screen readers?

**Suggestion**
- Add a [`<caption>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption) to the table
- Use [`scope` attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/columnheader_role#examples) on the `<th>` elements
- For the URL, add an [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) to provide more context.